opensrp-core/src/test/assets/app.properties

```diff
@@ -4,3 +4,4 @@ SHOULD_VERIFY_CERTIFICATE=false
 SYNC_DOWNLOAD_BATCH_SIZE=10
 system.toaster.centered=false
 allow.offline.login.with.invalid.token=false
+monitor.height=true
```

Comment:
This property may be misleading since it's implementation is in growth monitoring. The property checker bits should default to `false` so we can skip the actual entry in properties